test/integration/test_mounted_in_rails_app.rb

```diff
@@ -16,4 +16,32 @@ def app
       assert last_response.ok?, "Received bad response: #{last_response.inspect}"
     end
   end
+
+  def test_show_with_known_job
+    job = OpenStruct.new
```

Comment:
I spent some time futzing with `Minitest::Mock` here, but it was being very persnickety about the # of times that methods would be called. In the end, I didn't care that much because we've got enough coverage like this.